Documentation/Docker-setup/jobmanager-entrypoint.sh

```
@@ -0,0 +1,133 @@
+DASHBOARD_JOB_CLASS=${DASHBOARD_JOB_CLASS}
+DASHBOARD_JOB_NAME=${DASHBOARD_JOB_NAME}
+# -----------------------------
+
+# Start JobManager in the background
+echo "Starting JobManager..."
+/docker-entrypoint.sh jobmanager &
+JM_PID=$!
+
+# Wait for JobManager to be ready
+echo "Waiting for JobManager to be ready..."
+for i in {1..30}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)